src/main.rs

```diff
@@ -41,6 +41,43 @@ fn convert_axis_angle_to_euler_xyz(rx: f64, ry: f64, rz: f64, angle: f64) -> (f6
 	return (euler_x, euler_y, euler_z);
 }
 
+/// Convert from Euler XYZ to AxisAngle.
+///
+/// All rotations are expected and returned as radians.
+///
+/// # Arguments
+///
+/// * `euler_rx` - The rotation around the X axis.
+/// * `euler_ry` - The rotation around the Y axis.
+/// * `euler_rz` - The rotation around the Z axis.
+///
+/// Known robots that use this format:
```

Comment:
```suggestion
/// Known robots that use Euler XYZ:
```
To specify; might be also good to change the rest of the comments to make it uniform